Plots/Line/NCL_text_add_1.py

```diff
@@ -0,0 +1,66 @@
+"""
+NCL_text_add_1.py
+===============
+This script illustrates the following concepts:
+   - Adding text to a plot using plot data coordinates
+   - Set the font size of text
+
+See following URLs to see the reproduced NCL plot & script:
+    - Original NCL script: https://www.ncl.ucar.edu/Applications/Scripts/text_add_1.ncl
+    - Original NCL plot: https://www.ncl.ucar.edu/Applications/Images/text_1_lg.png
+"""
+
+###############################################################################
+# Import packages:
+
+import numpy as np
+import xarray as xr
+import matplotlib.pyplot as plt
+
+import geocat.datafiles as gdf
+from geocat.viz import util as gvutil
+
+###############################################################################
+# Read in data:
+
+# Open a netCDF data file using xarray default engine and load the data into xarrays
+ds = xr.open_dataset(gdf.get("netcdf_files/uv300.nc"))
+# Extract variables
+uz = ds.U.isel(time=0, lon=8)
+lon = ds.U.isel(time=0, lon=8).lat
+
+###############################################################################
+# Plot:
+
+# Generate figure (set its size (width, height) in inches) and axes
+plt.figure(figsize=(6.5, 6.5))
+ax = plt.gca()
+
+# Plot data
+plt.plot(lon, uz.values, c='gray')
```

Comment:
I think you should bump your linewidth down just a hair here to more closely replicate the original.